src/Microsoft.AspNetCore.Server.HttpSys/Constants.cs

```diff
@@ -13,6 +13,7 @@ internal static class Constants
         internal const string Close = "close";
         internal const string Zero = "0";
         internal const string SchemeDelimiter = "://";
+        internal const int Status101SwitchingProtocols = 101;
```

Comment:
Kestrel could use it too: https://github.com/aspnet/KestrelHttpServer/search?utf8=%E2%9C%93&q=101